tests/dftd3/dft-bench-interaction/input.dat

```diff
@@ -0,0 +1,61 @@
+#! functionals + DFTD3 interaction energies vs. other programs
+
+molecule h2o_dimer {
+0 1
+O  -1.551007  -0.114520   0.000000
+H  -1.934259   0.762503   0.000000
+H  -0.599677   0.040712   0.000000
+--
+0 1
+O   1.350625   0.111469   0.000000
+H   1.680398  -0.373741  -0.758561
+H   1.680398  -0.373741   0.758561
+no_reorient
+}
+
+set {
+   basis 6-31G
+   dft_radial_points 200
+   dft_spherical_points 590                
+   guess sad
+   e_convergence 9 
+   d_convergence 9 
+}
+
+qchem_data = {                              #TEST
+     'B97-D3':  -0.014457017899999869,      #TEST
+   'wB97X-D3':  -0.014866630699998495,      #TEST
+}                                           #TEST
+
+# Current version of Psi4 does not match Q-Chem for these tests
+expected_fail_qchem = ['wB97X-D3'] #TEST
+
+# The following data is from an older 
+# version of Psi4 (~April 2017, SHA: 53e752c)  
+old_psi4_data = {                           #TEST 
+    'B97-D3': -0.014455902448531788,        #TEST
+}                                           #TEST
+
+# Current version of Psi4 does not match pre-LibXC Psi4
+# (~April 2017, SHA: 53e752c)
+expected_fail_old_psi4 = [] #TEST
+
+
+for func in qchem_data.keys():                                                                                 #TEST
```

Comment:
yeah, so the dft-bench and python-vibanalysis tests simply can't go as they are (longest takes 40 min single thread). Sensible thing to do is to convert to pytest so that each fctl can be independent (and not need separation into plain and dftd3-needing. but pytest tests don't get run through ctest.

So longer-range plan is to convert all tests away from ctest and toward pytest, with the existing `psi4 input.dat` tests run still as executables just as p4n scripts are. There is a plugin (https://docs.pytest.org/en/3.0.0/xdist.html) that allows parallel execution. This is another motive behind #943. Would be glad of any feedback towards this plan.

Reply:
PyTest without modification will use a single instance of Psi4 for all the tests on a single thread. Im not quite sure we are ready to do that yet. Also if we covert these all to Python functions how would we handle examples? Just two immediate questions that pop to my mind.

Reply:
Out of curiosity is there a good driving factor to push towards PyTest?

Reply:
- ability to test installed software, not just source (this is the big one for me)
- easier to do conditions (addon present, version present, subsets)
- easier to share testing code (one file != one test)